pkg/admission/imagepolicy/imagepolicy_test.go

```diff
@@ -431,14 +483,6 @@ func TestAdmissionResolution(t *testing.T) {
 		pod.Spec.Containers[1].Image != "myregistry.com/mysql/mysql@sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4" {
 		t.Errorf("unexpected image: %#v", pod)
 	}
-
```

Comment:
Basically from now on the mutation error will never occur. Nevertheless I would keep the `mutationPreventer` in the Validate function in case some additional behaviour gets introduced. But now we can't test this hypothetical use case